src/components/button/button.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import {ButtonPrimary} from "./button"
+
+export default {
+  title: "Button",
+  component: ButtonPrimary,
+};
+
+export const ButtonStory = () => {
+  return (
+    <ButtonPrimary>
```

Comment:
I would go ahead and add all the other buttons here, so when storybook is pulled up all the buttons appear! Something like this: ``` <>
      <ButtonPrimary />
      <ButtonSecondary />
      <ButtonText />
      <ButtonIcon />
    </>```